dask_sql/physical/rex/core/call.py

```diff
@@ -613,6 +614,37 @@ def extract(self, what, df: SeriesOrScalar):
             raise NotImplementedError(f"Extraction of {what} is not (yet) implemented.")
 
 
+class ToTimestampOperation(Operation):
+    def __init__(self):
+        super().__init__(self.to_timestamp)
+
+    def to_timestamp(self, df, format):
+        # Remove double and single quotes from string
+        format = format.replace('"', "")
+        format = format.replace("'", "")
+
+        # TODO: format timestamps for GPU tests
+        if "cudf" in str(type(df)):
+            if format != "%Y-%m-%d %H:%M:%S":
+                print("Formatting timestamps not supported on GPU")
```

Comment:
based on my tests the gpu case fails for all cases when using `dd.to_datetime` and there's an additional bug with using some formats with `cudf.to_datetime`. It would be good to confirm that and replace this section with a error instead of a print.